### DISCUSSION TITLE
Totally broken

### DISCUSSION THREAD

**Discussion by Lamparter** [IviriusCommunity/Rebound#80] [General]
I can't contribute to Rebound anymore due to it being completely broken. Project builds fail with no given reason, WinRT references disappear, WinAppSDK problems, CsWinRT doesn't work, and a whole load of other problems causing me to be totally unable to work on it. 

I even went as far as to reinstall visual studio and the SDKs but nothing works. Goodbye for now ❤️

**Comment by jarno9981** [comment]
> I can't contribute to Rebound anymore due to it being completely broken. Project builds fail with no given reason, WinRT references disappear, WinAppSDK problems, CsWinRT doesn't work, and a whole load of other problems causing me to be totally unable to work on it. 
> 
> I even went as far as to reinstall visual studio and the SDKs but nothing works. Goodbye for now ❤️

Rebuild full project then update stable sdk and build works for me

**Reply by Lamparter** [reply]
I'll try that but chances are it's unlikely to make a difference

**Comment by IviriusMain** [comment]
Works now 😌
